parm/config/config.base.emc.dyn

```diff
@@ -281,15 +280,9 @@ export WRITE_DOPOST=".true."
 export WRITE_NSFLIP=".true."
 
 # suffix options depending on file format
-if [ $OUTPUT_FILE = "netcdf" ]; then
-    export SUFFIX=".nc"
-    export NEMSIO_IN=".false."
-    export NETCDF_IN=".true."
-else
-    export SUFFIX=".nemsio"
-    export NEMSIO_IN=".true."
-    export NETCDF_IN=".false."
-fi
+export SUFFIX=".nc"
+export NEMSIO_IN=".false."
+export NETCDF_IN=".true."
```

Comment:
Should be able to get rid of these variables entirely now. Find wherever they are used and replace with hard-coded values.